requirements/build-pkg.txt

```diff
@@ -1,3 +1,3 @@
 twine
 wheel
-build
+build
```

Comment:
Looks like this was accidentally committed.

Reply:
Should be now fixed.